io.freetubeapp.FreeTube.yml

```diff
@@ -11,6 +11,7 @@ finish-args:
   - --device=dri
   - --share=ipc
   - --socket=x11
+  - --socket=wayland
```

Comment:
x11 should be replaced by fallback-x11

Reply:
This should rather be only `--socket=x11` @PrestoN

Wayland was supposed to be opt in so that users can experiment with it by using flatpak run --socket=wayland or setting a override

Reply:
See the docs here https://docs.flatpak.org/en/latest/electron.html#sandbox-permissions

> To enable experimental native Wayland support in Electron>=20, the --ozone-platform-hint=auto flag can be passed to the program. auto will choose Wayland when the session is wayland and Xwayland or X11 otherwise.

The recommended option is to leave it to the user. So --socket=x11 should be used in manifest and Wayland can be tested with:

`flatpak run --socket=wayland org.flathub.el...`

Reply:
See https://github.com/flathub/io.freetubeapp.FreeTube/pull/102